zetaclient/tss/service_test.go

```diff
@@ -0,0 +1,241 @@
+package tss_test
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/rs/zerolog"
+	"github.com/samber/lo"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/zeta-chain/node/cmd"
+	"github.com/zeta-chain/node/pkg/cosmos"
+	"github.com/zeta-chain/node/zetaclient/testutils/mocks"
+	"github.com/zeta-chain/node/zetaclient/tss"
+	"gitlab.com/thorchain/tss/go-tss/blame"
+	tsscommon "gitlab.com/thorchain/tss/go-tss/common"
```

Comment:
I think we could look into replacing all `gitlab.com/thorchain/tss/go-tss/common` into the ZetaChain fork and remove the replace statement in `go.mod`

Reply:
**UPD**: I don't think it's feasible in this PR because go-tss fork has origin's package name: https://github.com/zeta-chain/go-tss/blob/37db84dd363f463e4b582e7b62a965d529c8d5ee/go.mod#L1